packages/babel-preset-sui/lib/is-installed.js

```diff
@@ -1,9 +1,15 @@
-module.exports = function (base, module) {
-  var isInstall = false
-  try {
-    require(base)
-    isInstall = true
-  } catch (e) {}
+module.exports = function (packagesToTest, moduleToRequire) {
+  // check if the param is an array, if not, convert to it
+  packagesToTest = Array.isArray(packagesToTest) ? packagesToTest : [packagesToTest]
```

Comment:
friki comment:

packagesToTest = [...packagesToTest] :P

Reply:
```
let packagesToTest = 'react'
packagesToTest = [...packagesToTest] 
console.log(packagesToTest) // ['r', 'e', 'a', 'c', 't']
```

Reply:
Ohhh sure, an string is an iterable :(

Reply:
hahaha :+1: Nice anyway!

Reply:
Fixed! :)